extensions/amp-youtube/0.1/amp-youtube.js

```diff
@@ -94,6 +119,40 @@ class AmpYoutube extends AMP.BaseElement {
   }
 
   /** @private */
+  getPlayerReadyPromise_() {
+    return new Promise(resolve => {
```

Comment:
The reason I wanted to keep this as its own method was to allow me to stub it for testing. Any idea how to best test this? Will look into it meanwhile.